android/app/src/main/java/org/fossasia/openevent/adapters/SessionsListAdapter.java

```diff
@@ -125,6 +125,12 @@ public SessionViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
     @Override
     public void onBindViewHolder(final SessionViewHolder holder, final int position) {
         Session session = getItem(position);
+
+        //removing session when in draft state
+        if(session.getState().equals("draft")) {
+            removeItem(session);
```

Comment:
remove() is not recognised. should i create a list variable while initializing and then use that?